pkgs/development/tools/build-managers/buildbot9/plugins.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv,
+  fetchurl,
+  buildPythonApplication,
+  python
+}:
+
+{
+  www = buildPythonApplication rec {
+    name = "buildbot-www-0.9.0rc1";
+    namePrefix = "";
+    format = "wheel";
+
+    src = fetchurl {
+      url = "https://pypi.python.org/packages/f4/3c/8bf1add762739d1a15d63c91140d7fdc32d872860229ffdb10e735fdaa7a/buildbot_www-0.9.0rc1-py2-none-any.whl";
+      sha256 = "21c3b55be0c1622757a04e6573e8503f2222ba80f01af3770eb04d71e3c5a8fb";
+    };
+
+    propagatedBuildInputs = [ python ];
```

Comment:
get a weird error without it about 'pkg_resources'. seems to be needed only for the wheel builds, though.